src/core/Table/Table.test.tsx

```diff
@@ -2373,3 +2373,184 @@ it('should not have `draggable` attribute on columns with `disableReordering` en
   expect(headerCells[3].getAttribute('draggable')).toBe('true'); // Description column
   expect(headerCells[4].getAttribute('draggable')).toBeFalsy(); // View column
 });
+
+it('should render empty action column with column manager', () => {
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+        {
+          id: 'description',
+          Header: 'Description',
+          accessor: 'description',
+        },
+        {
+          id: 'view',
+          Header: 'View',
+          Cell: () => 'View',
+        },
+        ActionColumn(true),
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns,
+    data: mockedData(),
+    isSelectable: true,
+    subComponent: (row) => (
+      <div>{`Expanded component, name: ${row.original.name}`}</div>
+    ),
+  });
+
+  const columnManager = container.querySelector('.iui-button') as HTMLElement;
+
+  expect(columnManager).toBeTruthy();
+  userEvent.click(columnManager);
+
+  expect(document.querySelector('.iui-menu')).toBeTruthy();
+
+  const columnManagerColumns = document.querySelectorAll('.iui-menu-item');
+  expect(columnManagerColumns[0].textContent).toBe('Name');
+  expect(columnManagerColumns[1].textContent).toBe('Description');
+  expect(columnManagerColumns[2].textContent).toBe('View');
+});
+
+it('should render action column with column manager', () => {
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+        {
+          id: 'description',
+          Header: 'Description',
+          accessor: 'description',
+        },
+        {
+          ...ActionColumn(true),
+          id: 'view',
+          Cell: () => 'View',
```

Comment:
Not sure how useful this test is.. You do not even check if Cell content appears.